src/main/resources/application.properties

```diff
@@ -1,7 +1,10 @@
 spring.application.name=hng-java-boilerplate
 
 # datasource configuration
-spring.datasource.url=jdbc:${connection_string}
+# spring.datasource.url=jdbc:postgresql://localhost:5432/postgres
```

Comment:
remove comments. local test changes should not be pushed

Reply:
changed application.properties back to default